### PR TITLE
Solve #69, error msg

### PR DESCRIPTION
**Pull request by SabineHaas**
Addresses issue #69.

### CODE REVIEW

File: windpowerlib/wind_farm.py [hunk 1]
```diff
@@ -402,8 +403,10 @@ def assign_power_curve(self, wake_losses_model='wind_farm_efficiency',
...
                         wind_farm_efficiency=self.efficiency))
             else:
-                logging.info("`wake_losses_model` is {} but wind farm ".format(
-                    wake_losses_model) + "efficiency is NOT taken into "
-                                         "account as it is None.")
+                msg = (
+                    "If you use `wake_losses_model` '{model}' your WindFarm "
+                    "needs an efficiency but `efficiency` of {farm} is {eff}.")
+                raise ValueError(msg.format(model=wake_losses_model, farm=self,
+                                            eff=self.efficiency))
```
Comment (uvchik) [inline windpowerlib/wind_farm.py]: The output of `farm` is quite long and people may miss the rest of the message. Is it possible to change the order?

Something like this:
```diff
- "If you use `wake_losses_model` '{model}' your WindFarm "
- "needs an efficiency but `efficiency` of {farm} is {eff}.")
+ "If you use `wake_losses_model` '{model}' your WindFarm "
+ "needs an efficiency but `efficiency` is {eff}.\n\n"
+ "Failing farm:\n {farm}")

Comment (SabineHaas) [reply windpowerlib/wind_farm.py]: Good point, thanks!

**Comment by SabineHaas** [comment]
I found another one of these messages.. nice to have when there is time: check for all messages.

**Comment by uvchik** [comment]
There was a merge conflict due to the stickler-PR I just merged but I fixed it. Sorry :smile:

**Comment by uvchik** [comment]
> I found another one of these messages.. nice to have when there is time: check for all messages.

I would not fix it here but rather open a new issue and merge this.

**Pull request merged**